README.md

```diff
@@ -74,9 +103,11 @@ pipx ensurepath
 pipx install tox
 ```
 
-3. Copy and paste `integration_test_project/example-env.sh` as `env.sh`. Fill in the missing values. Source the file with `. ./env.sh`.
+3. From this repository, copy and paste the `integration_test_project/example-env.sh` file and save as `env.sh` in your local repo. Fill in the missing values.
```

Comment:
```suggestion
3. Copy and paste the `integration_test_project/example-env.sh` file and save as `env.sh`. Fill in the missing values.
```

Think the confusion here may be that I only added the example-env.sh file yesterday, but it will get cloned for all future users who want to contribute